src/Controller/Admin/ConfigController.php

```diff
@@ -32,6 +34,7 @@ public function edit(Request $request): Response
         $form = $this->createForm(ConfigType::class, $this->config->getAll());
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
+            $this->handleTechnicalEmail((string) $form->get(Config::TECHNICAL_EMAIL)->getData());
```

Comment:
I'm not sure. In that case it will be possible to change this config and bypass the logic of sending the technical email to the server, it would be better to add it to the handler (`ChangeConfigHandler`).